docs/examples/example1.py

```diff
@@ -39,6 +39,16 @@ def previewWidget(self, libraryWidget):
         """
         return ImagePreviewWidget(self)
 
+    @staticmethod
+    def isPathSuitable(self, path):
```

Comment:
This looks good. 

It might be better to name the method to "isPathValid".  This will make the name more consistent with the QT naming style.

Also, can we make this change a separate pull request?

Thanks